### PR TITLE
[REVIEW] Model.py and Config file for HugeCTR

### PR DESCRIPTION
**Pull request by oyilmaz-nvidia**
This PR includes the model.py and config file for HugeCTR. Since HugeCTR's input structure is different than TF, we need a separate model.py and config file for hugectr. 

Ensemble model (NVTabular + HugeCTR) will be in a separate PR.

### CODE REVIEW

File: nvtabular/inference/triton/__init__.py [hunk 1]
```diff
@@ -152,26 +174,73 @@ def convert_triton_output_to_df(columns, response):
...
 
-    for column, dtype in workflow.input_dtypes.items():
-        config.input.append(
-            model_config.ModelInput(name=column, data_type=_convert_dtype(dtype), dims=[-1, 1])
+    if max_batch_size is None:
+        config = model_config.ModelConfig(name=name, backend="python")
+    else:
+        config = model_config.ModelConfig(
+            name=name, backend="python", max_batch_size=max_batch_size
+        )
```
Comment (benfred) [inline nvtabular/inference/triton/__init__.py:188]: I don't think we need the if statement here (like the default value for max_batch_size is None inside ModelConfig)
```suggestion
    config = model_config.ModelConfig(name=name, backend="python", max_batch_size=max_batch_size)
```

**Review by benfred — APPROVED** [review]
Looks good - thanks!

**Comment by jperez999** [comment]
rerun tests

**Comment by jperez999** [comment]
rerun tests

**Comment by jperez999** [comment]
rerun tests

**Comment by benfred** [comment]
rerun tests

**Pull request merged**